recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
I was wrong, X packages are linked via `libcairo.so` and are present upstream. Although I think such dependency is out of conda's scope. 

Those packages are available only in `conda-forge` repo. Is it now a "standard" dependency for bioconda channel?